packages/reporter/cypress/integration/shortcuts_spec.ts

```diff
@@ -135,5 +135,15 @@ describe('shortcuts', function () {
       cy.get('button.stop').trigger('mouseover')
       cy.get('.cy-tooltip').should('have.text', 'Stop Running S')
     })
+
```

Comment:
@majidzeno - I went ahead and pushed a test for these changes onto your branch, this is what @jennifer-shehane was asking for.

Looks good to me. :+1: